### PR TITLE
feat: US138564 Add tests for d2l-questions-written-response

### PR DESCRIPTION
**Pull request by tdevos-d2l**
[US138564](https://rally1.rallydev.com/#/?detail=/userstory/632753245761&fdp=true): [Consistent Evaluation][Quizzes][UI][Questions] Add tests for d2l-questions-written-response

- Adding testing to the `d2l-questions-written-reponse` component
- Adding Axe tests to  written response presentational
- Making v-diff tests test the written response presentational component directly
- Moved written response's data loading into it's own function so it can send an event when finished loading
   - This helps the tests know when data is loaded
   - Also needed for skeleton work. See: [US138584](https://rally1.rallydev.com/#/?detail=/userstory/632832032003&fdp=true)

### CODE REVIEW

File: components/d2l-questions-written-response-presentational.js [hunk 1]
```diff
@@ -167,7 +167,7 @@ class D2lQuestionWrittenResponsePresentational extends LocalizeQuestions(LitElem
 
 	_renderResponse() {
 		//Checking for text or length so we can display "0 words" when a response exists but is empty
-		if (this.readonly && this.responseText || this.responseLength) {
+		if (this.readonly && this.responseText !== undefined || this.responseLength) {
```
Comment (tdevos-d2l) [inline components/d2l-questions-written-response-presentational.js:170]: Without this, the empty response v-diff test was failing. When `this.responseText` is undefined is when we have no response: https://github.com/BrightspaceHypermediaComponents/questions/blob/d77769ea40ffee26c9ede751c384f1a10e9e93e2/components/d2l-questions-written-response.js#L84
So checking if undefined should be okay. Let me know if there was another reason for this though.

**Review by JamieTran — APPROVED** [review]
lgtm

File: components/d2l-questions-written-response-presentational.js (hunk 1 above)
Comment (maddiegabriel) [reply components/d2l-questions-written-response-presentational.js:170]: makes sense to me

File: test/d2l-questions-written-response-presentational-data.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+export const questionText = '<p>Random Question</p>';
+export const responseText = '<p>Chocolate muffin sugar plum muffin cupcake</p>';
```
Comment (maddiegabriel) [inline test/d2l-questions-written-response-presentational-data.js:2]: mmmmmmm

**Review by maddiegabriel — APPROVED** [review]
great!

**Pull request merged**